data-science-onramp/data-cleaning/clean_test.py

```diff
@@ -109,7 +106,7 @@ def setup_and_teardown_cluster():
         project_id=PROJECT_ID,
         region=CLUSTER_REGION,
         cluster_name=DATAPROC_CLUSTER,
-        timeout=300
+        timeout=300,
```

Comment:
This doesn't hurt but you shouldn't need to do this. Were there timeout errors deleting the cluster?

Reply:
leftover from Tushar - can remove if need be

Reply:
Removing is fine unless it somehow breaks

Reply:
Will remove next, just testing locally now

Reply:
local tests passed, just pushed removal